scripts/build

```diff
@@ -5,5 +5,5 @@ source $(dirname $0)/version
 
 cd $(dirname $0)/..
 mkdir -p bin
-echo Building submariner-engine version $VERSION
-CGO_ENABLED=0 go build -ldflags "-X main.VERSION=$VERSION" -o bin/submariner-engine main.go
\ No newline at end of file
+echo Building coastguard version $VERSION
```

Comment:
Why?, I want CI to make sure the code I'm merging at least compiles.

Reply:
then do a PR that enables the testing before this PR ;)

Reply:
I can't because there wasn't anything I could build, I could create an scaffold on such PR.

But doing such work doesn't seem to provide much in return at this point of the project, so I prefer to keep it on this one.